appinventor/blocklyeditor/src/generators/yail/componentblock.js

```diff
@@ -332,3 +332,17 @@ Blockly.Yail.component_component_block = function() {
   return [Blockly.Yail.YAIL_GET_COMPONENT + this.getFieldValue("COMPONENT_SELECTOR") + Blockly.Yail.YAIL_CLOSE_COMBINATION,
           Blockly.Yail.ORDER_ATOMIC];
 };
+
+/**
+ * Returns a function that takes no arguments, generates Yail to get a list of all
+ * components of a type, and returns a 2-element array containing the component
+ * getter code and the operation order Blockly.Yail.ORDER_ATOMIC.
+ *
+ * @param {String} instanceName
+ * @returns {Function} component getter code generation function with instanceName bound in
+ */
+Blockly.Yail.component_all_component_block = function() {
```

Comment:
For forwards compatibility, please switch this to `Blockly.Yail['component_all_component_block']`